quickstarts/blazor-web-assembly/config.yml

```diff
@@ -0,0 +1,53 @@
+
+
+slug: blazor-web-assembly
+description: |
+  ## Comprehensive monitoring quickstart for Blazor Web Assembly
+
+  Microsoft developed Blazor Web Assembly to build web UIs with C#, HTML, and CSS. As a feature of ASP.NET, our Blazor Web Assembly integration with the Browser Monitoring agent lets you monitor your apps with our quickstart dashboards right out of the box.
+
+
+  ## Why monitor Blazor Web Assembly?
```

Comment:
The text in this section is all about why I should Blazor WebAssembly *not* New Relic. This whole section needs to be rewritten to answer the question "Why should I monitor my Blazor WebAssembly app with New Relic?"